rangebar/src/com/edmodo/rangebar/Bar.java

```diff
@@ -127,6 +127,10 @@ int getNearestTickIndex(Thumb thumb) {
 
         final int nearestTickIndex = (int) ((thumb.getX() - mLeftX + mTickDistance / 2f) / mTickDistance);
 
+        if (nearestTickIndex < 0) return 0;
+
+        if (nearestTickIndex > mNumSegments) return mNumSegments;
```

Comment:
Is `mNumSegments` the right value to use for max?  Or should it be `mNumSegments - 1`?  (Haven't looked to check, just wondering)

The important thing is that the value returned from here must be a value index given the bar's normal tick range.